.travis.yml

```diff
@@ -57,6 +57,18 @@ script:
 - ./test.sh
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then PATH=$PATH:$(pwd) ./test_e2e.sh; fi
 
+jobs:
+  include:
+    - stage: Go Coverage
+      before_script:
+        - go get github.com/mattn/goveralls
```

Comment:
What is the relationship between this tool and `coveralls.io`?

Reply:
it is used for we do the integration. The module allows us sent the coverage-all.out [here](https://github.com/kubernetes-sigs/kubebuilder/pull/1207/files#diff-354f30a63fb0907d4ad57269548329e3R70).  I will add a comment to make clear.

Reply:
can we pin this to a particular version, please?